vm-setup-script-usage.md

```diff
@@ -0,0 +1,92 @@
+# Orka VM Setup Script Usage
+## This script automates the configuration and setup of macOS Orka VMs for optimal performance. It configures system settings, installs required tools, and prepares the VM for use with Orka.
+
+### System requirements:
+
+- macOS IPSW file (Tahoe, Sequoia, or Sonoma)
+- Admin/sudo privileges on the VM
+- Internet connection for downloading packages
+- Terminal access (via SSH, Screen Sharing, or direct console)
+
+### Before running:
+
+- Ensure you have administrator access to the VM
+- Close any important applications (they will be terminated during cleanup)
+- Save any work in progress (the VM will reboot automatically)
+
+### Installation:
+
+#### Download via GitHub
+
+- ```/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/macstadium/orka-images/orka-ipsw-setup.sh)"```
+- ```chmod +x orka-ipsw-setup.sh```
+- ```./orka-ipsw-setup.sh```
+
+### Installation steps when running the script:
+
+#### On MacOS Sequoia:
+
+- When prompted for Terminal permissions access, click 'Accept'
+- Terminal will need full disk access to run the script. Configure this by going to your System Settings -> Privacy and Security -> Full Disk Access and adding 'Terminal' by clicking the + button and searching for the app in the search bar.
+- Enter your password when prompted during the script installation process
+- When the script shell displays ```sh-3.2#``` type ```exit``` and press ```Enter``` to continue script installation
```

Comment:
Also - If we do not need multiline formatted text, or do not care about rendering highlighted syntax, then it is common to just use single backticks. For example this should be

````
When the script shell displays `sh-3.2#` type `exit` and press `Enter` to continue script installation
````

Which renders as:

When the script shell displays `sh-3.2#` type `exit` and press `Enter` to continue script installation

Reply:
fixed